ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -242,8 +242,14 @@ func (scbuilder *SiteConfigBuilder) getClusterCR(clusterId int, siteConfigTemp S
 			}
 			mapIntf[k] = value
 		} else if reflect.ValueOf(v).Kind() == reflect.String &&
-			strings.HasPrefix(v.(string), "siteconfig.") {
-			valueIntf, err := siteConfigTemp.GetSiteConfigFieldValue(v.(string), clusterId, nodeId)
+			strings.HasPrefix(v.(string), "{{") &&
+			strings.HasSuffix(v.(string), "}}") {
+			// We can be cleaner about this, but this translation is minimally invasive for 4.10:
+			key, err := translateTemplateKey(v.(string))
```

Comment:
Here's the only change to the parsing logic!  That and the fairly simple (and well-tested) `translateTemplateKey` function below.